src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -249,6 +318,9 @@ void NSModel<SortPolicy>::Serialize(Archive& ar,
                                     const unsigned int /* version */)
 {
   ar & data::CreateNVP(treeType, "treeType");
+  ar & data::CreateNVP(leafSize, "leafSize");
+  ar & data::CreateNVP(tau, "tau");
+  ar & data::CreateNVP(rho, "rho");
```

Comment:
Looks good.  I noticed that the previous revision did not serialize `leafSize`, so this fixes a bug too!